modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/info/item/provider/test/DDMFormValuesInfoFieldValuesProviderTest.java

```diff
@@ -291,6 +387,22 @@ private DDMFormField _createDDMFormField(
 		return ddmFormField;
 	}
 
+	private String _read(String fileName) throws Exception {
+		Class<?> clazz = getClass();
+
+		String file = StringUtil.read(
+			clazz.getClassLoader(),
+			"com/liferay/dynamic/data/mapping/dependencies/" + fileName);
+
+		if (fileName.contains("xsd")) {
+			return file;
+		}
+
+		JSONObject jsonObject = JSONFactoryUtil.createJSONObject(file);
+
+		return jsonObject.toString();
+	}
```

Comment:
Here we have some problems. Since the signature to this package is the same as the one I'm changing in InfoFieldValuesTest class, the code doesn't understand I want to look for a DDMBaseTest in another folder and try to find in the current visible worktree, returning me a non-existing Class. In order to solve this, I extended from the BaseServiceTest class. Could you please take a look?